packages/persistent-ts/package.json

```diff
@@ -3,21 +3,16 @@
   "version": "0.19.1",
   "description": "Persistent data structures for TypeScript.",
   "type": "module",
-  "main": "./lib/cjs/index.js",
-  "module": "./lib/index.js",
+  "main": "./lib/index.js",
```

Comment:
This package needs subpath exports for the as-sha256 hasher and the setHasher function, see https://github.com/ChainSafe/lodestar/pull/7387/files

Reply:
I wanted to add extra named exports in a separate PR to have clarity in the change logs. What do you think?